docs/src/appendix/fractional_step.md

```diff
@@ -1,44 +1,45 @@
 # Fractional step method
 
-Solving the momentum equation \eqref{eq:momentumFV} coupled with the continuity equation \eqref{eq:continuityFV} can be
-cumbersome so instead we employ a fractional step method. To approximate the solution of the coupled system we first
-solve an approximation to the discretized momentum equation \eqref{eq:momentumFV} for an intermediate velocity field
-``\bm{u}^\star`` without worrying about satisfying the incompressibility constraint. We then project ``\bm{u}^\star`` onto
-the space of divergence-free velocity fields to obtain a value for ``\bm{u}^{n+1}`` that satisfies
-\eqref{eq:continuityFV}.
+Solving the momentum equation \eqref{eq:momentumFV} coupled with the continuity equation can 
```

Comment:
I see this is where the referencing to equations does not work.  I am fine with having this fixed in another PR if that's easier.